# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/shadow/login_defs.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./etc/login.defs.orig	Fri Jan  5 15:33:49 2001
+++ ./etc/login.defs		Fri Jan  5 15:38:52 2001
@@ -230,7 +230,7 @@
 #
 # If compiled with cracklib support, where are the dictionaries
 #
-CRACKLIB_DICTPATH	/var/cache/cracklib/cracklib_dict
+CRACKLIB_DICTPATH	/usr/share/dict/cracklib_dict
 
 #
 # Min/max values for automatic uid selection in useradd
@@ -299,7 +299,7 @@
 # Set to "no" if you need to copy encrypted passwords to other systems
 # which don't understand the new algorithm.  Default is "no".
 #
-#MD5_CRYPT_ENAB	no
+MD5_CRYPT_ENAB	yes
 
 #
 # List of groups to add to the user's supplementary group set
@@ -357,7 +357,7 @@
 # Setting GETPASS_ASTERISKS to -1 reverts to the traditional getpass()
 # without any new features.  This is the default.
 #
-#GETPASS_ASTERISKS 1
+GETPASS_ASTERISKS 3
 
 #
 # Enable setting of the umask group bits to be the same as owner bits
